src/main/data/vcf.js

```diff
@@ -41,12 +43,28 @@ function extractLocusLine(vcfLine: string): LocusLine {
 
 function extractVariant(vcfLine: string): Variant {
   var parts = vcfLine.split('\t');
+  var frequency = null;
+  if (parts.length>=7){
```

Comment:
What if this is a multi-sample VCF and the first sample is not relevant for the user? I am asking this; because I know that mutect can sometimes order the samples in a way that  the normal sample comes before the tumor one.